next-app/src/utils/hooks/useColorMode.ts

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+
+import { COLOR_MODE } from 'constants/colorMode'
+
+export const useColorMode = () => {
+  const [isDarkMode, setIsDarkMode] = useState(false)
+
+  useEffect(() => {
+    const preferredColorMode = localStorage.getItem(`${COLOR_MODE}`)
```

Comment:
```suggestion
    const preferredColorMode = localStorage.getItem(COLOR_MODE)
```
템플릿 리터럴 안써도 되지 않을까요?